{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:4:7\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"problem","name":"Error","stack":"Error: problem\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/get-all-users.js:1:479\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:47:12)"}
{"level":"error","message":"This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\".","name":"UnhandledPromiseRejection","stack":"UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason \"undefined\"."}
{"level":"error","message":"","name":"Error","stack":"Error\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/update-user.js:1:507\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}
{"level":"error","message":"","name":"Error","stack":"Error\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/Users/update-user.js:1:569\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}
{"level":"error","message":"The \"chunk\" argument must be of type string or an instance of Buffer or Uint8Array. Received an instance of Object","name":"TypeError","stack":"TypeError [ERR_INVALID_ARG_TYPE]: The \"chunk\" argument must be of type string or an instance of Buffer or Uint8Array. Received an instance of Object\n    at new NodeError (node:internal/errors:371:5)\n    at write_ (node:_http_outgoing:742:11)\n    at ServerResponse.end (node:_http_outgoing:855:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:260\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:144:13)\n    at Route.dispatch (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/route.js:114:3)\n    at Layer.handle [as handle_request] (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/express/lib/router/index.js:346:12)"}
{"level":"error","message":"allUsers is not iterable","name":"TypeError","stack":"TypeError: allUsers is not iterable\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:310)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"either setProtectedHeader or setUnprotectedHeader must be called before #sign()","name":"JWSInvalid","stack":"JWSInvalid: either setProtectedHeader or setUnprotectedHeader must be called before #sign()\n    at FlattenedSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/flattened/sign.js:34:19)\n    at CompactSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/compact/sign.js:14:43)\n    at SignJWT.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jwt/sign.js:22:20)\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:465)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"either setProtectedHeader or setUnprotectedHeader must be called before #sign()","name":"JWSInvalid","stack":"JWSInvalid: either setProtectedHeader or setUnprotectedHeader must be called before #sign()\n    at FlattenedSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/flattened/sign.js:34:19)\n    at CompactSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/compact/sign.js:14:43)\n    at SignJWT.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jwt/sign.js:22:20)\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:465)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"either setProtectedHeader or setUnprotectedHeader must be called before #sign()","name":"JWSInvalid","stack":"JWSInvalid: either setProtectedHeader or setUnprotectedHeader must be called before #sign()\n    at FlattenedSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/flattened/sign.js:34:19)\n    at CompactSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/compact/sign.js:14:43)\n    at SignJWT.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jwt/sign.js:22:20)\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:471)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"Key must be one of type KeyObject or CryptoKey.","name":"TypeError","stack":"TypeError: Key must be one of type KeyObject or CryptoKey.\n    at asymmetricTypeCheck (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/lib/check_key_type.js:17:15)\n    at checkKeyType (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/lib/check_key_type.js:44:9)\n    at FlattenedSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/flattened/sign.js:55:41)\n    at CompactSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/compact/sign.js:14:43)\n    at SignJWT.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jwt/sign.js:22:20)\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:506)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"Key must be one of type KeyObject or CryptoKey. Received an instance of Uint8Array","name":"TypeError","stack":"TypeError: Key must be one of type KeyObject or CryptoKey. Received an instance of Uint8Array\n    at asymmetricTypeCheck (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/lib/check_key_type.js:17:15)\n    at checkKeyType (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/lib/check_key_type.js:44:9)\n    at FlattenedSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/flattened/sign.js:55:41)\n    at CompactSign.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/compact/sign.js:14:43)\n    at SignJWT.sign (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jwt/sign.js:22:20)\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:505)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"Invalid or unsupported JWK \"alg\" (Algorithm) Parameter value","name":"JOSENotSupported","stack":"JOSENotSupported: Invalid or unsupported JWK \"alg\" (Algorithm) Parameter value\n    at generateSecret (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/runtime/generate.js:33:19)\n    at generateSecret (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/key/generate_secret.js:6:45)\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:540)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"Invalid or unsupported JWK \"alg\" (Algorithm) Parameter value","name":"JOSENotSupported","stack":"JOSENotSupported: Invalid or unsupported JWK \"alg\" (Algorithm) Parameter value\n    at generateSecret (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/runtime/generate.js:33:19)\n    at generateSecret (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/key/generate_secret.js:6:45)\n    at loginService (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/services/Users/login.js:1:539)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/routers/login.js:1:298"}
{"level":"error","message":"signature verification failed","name":"JWSSignatureVerificationFailed","stack":"JWSSignatureVerificationFailed: signature verification failed\n    at flattenedVerify (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/flattened/verify.js:84:15)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async compactVerify (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jws/compact/verify.js:18:22)\n    at async jwtVerify (/Users/Mikhail_Proshin/work/NodeJS_Course/node_modules/jose/dist/node/cjs/jwt/verify.js:9:22)\n    at async checkAuthorisation (/Users/Mikhail_Proshin/work/NodeJS_Course/dist/HomeWork_2/middlewares/check-authorisation.js:1:216)"}
